modules/ROOT/pages/changelog.adoc

```diff
@@ -4,6 +4,91 @@
 
 NOTE: This is the {productname} Community version changelog. For information about the latest {cloudname} or {enterpriseversion} Release, see: xref:release-notes.adoc[{productname} Release Notes].
 
+== 6.4 - 2023-03-15
+
+=== Added
+* New `tree` component that can be used in dialog body panel.
+* renderUI property in the `Theme` type can now return a `Promise<RenderResult>` instead of `RenderResult`.
+* New `isEditable` API to `editor.selection` that returns true or false if the current selection is editable.
+* New `isEditable` API to `editor.dom` that returns true or false if the specified node is editable.
+* New `setText` and `setIcon` methods added to menu button and toolbar button apis.
+* New `highlight_on_focus` option which enables highlighting the content area on focus.
+* New `fontsizeinput` toolbar item which allows the user to set the size via input and also increase and decrease it with `+` and `-` buttons.
+* Added `skipFocus` option to the `ToggleToolbarDrawer` command to preserve focus.
+* New `font_size_input_default_unit` option allows entry of numbers without a unit in `fontsizeinput`. They are then parsed as the set unit. If `font_size_input_default_unit` is not set the default is `pt`.
+* New `group` and `togglebutton` in view.
+* New `togglebutton` in dialog footer buttons.
+* Added `toggleFullscreen` to dialog API.
```

Comment:
Not clear what these new things are

Reply:
They are the same new things as are in changelog.md in the TinyMCE repo. As in [these three lines](https://github.com/tinymce/tinymce/blob/db45bd16f11947b587fede0ec1b9745adc609e30/modules/tinymce/CHANGELOG.md?plain=1#L19)